### DISCUSSION TITLE
Is Sunshine able to Stream a 1440p monitor to a 4k TV?

### DISCUSSION THREAD

**Discussion by fxzzi** [LizardByte/community#382] [Sunshine] [answered]
Sunshine can obviously do it at 1440p, but what can I do on Arch Linux / Hyprland to be able to stream at 4k even though the monitor plugged into my PC is not 4k, but 1440p?

**Comment by Barina** [comment, marked as answer]
Yes. read here:
https://www.reddit.com/r/MoonlightStreaming/comments/rzpcpc/moonlight_streaming_without_monitor_no_dummy_plug/

I've done that and everything works well. (although it is pain to enable\disable the dummy monitor every time without a script)
